feature_extract_enable : 0
point_filter_num : 2
max_iteration : 10
dense_map_enable : 1
cut_points_above_ground: true
filter_size_surf : 0.15
filter_size_map : 0.3
filter_size: 0.2
down_sample_manner: 1
cube_side_length : 20
debug : 0
img_scale : 1.0
grid_size : 40
patch_size : 8
depth_search_radius: 4
img_enable : 1
lidar_enable : 1
trans_thre: 0.01
rot_thre: 0.1309
equalize: false
outlier_threshold : 3000 # 
score_threshold: 100
ncc_en: false
ncc_thre: 0
window_size : 20
img_point_cov : 10000.0 # 1000
line_cov : 10.0 # 1000
triangulate_point_cov: 10.0
enable_triangulate: true
enable_line: true
enable_BA: false
laser_point_cov : 0.001 # 0.001
cam_fx: 863.4241
cam_fy: 863.4171
cam_cx: 640.6808
cam_cy: 518.3392
read_enable: true
pub_pcl: true
encoding: "bgr8"
map_dir: "/home/i/vio_in_lidar_map/src/vio_in_lidar_map/map/scans_r3_0.pcd"
path_file: "path_r3_0.txt"
lines_file: "line_r3_0.txt"
init_num: 100
pure_imu: false
common:
    lid_topic:  "/livox/lidar"
    imu_topic:  "/livox/imu"

preprocess:
    lidar_type: 1 # Livox Avia LiDAR
    scan_line: 6
    blind: 5 # blind x m disable

mapping:
    acc_cov_scale: 100
    gyr_cov_scale: 10000
    acc_Q_scale: 10
    gyr_Q_scale: 10
    fov_degree:    90
    extrinsic_T: [ 0.04165, 0.02326, -0.0284 ]
    extrinsic_R: [ 1, 0, 0,
                   0, 1, 0,
                   0, 0, 1]

camera:
    # img_topic: /usb_cam/image_raw
    # img_topic:  /camera/image_color
    img_topic: /left_camera/image
    #xiyuan
    Rcl: [ -0.0011, -1.0000, 0.0005,
            -0.0159,-0.0005,-0.9999,
            0.9999,-0.0011,-0.0159]
    Pcl: [0.0475, -0.0304, -0.0506]
    blind: 3
